tests/wamr-test-suites/test_wamr.sh

```diff
@@ -856,12 +812,73 @@ function collect_coverage()
     fi
 }
 
+# decide whether execute test cases in current running mode based on the current configuration or not
+# return 1 if the test case should be executed, otherwise return 0
+function do_execute_in_running_mode()
+{
+    local RUNNING_MODE="$1"
+
+    # wasm64(memory64) is only enabled in interp and aot mode(?)
```

Comment:
memory64 will be enabled in aot mode soon, after PR https://github.com/bytecodealliance/wasm-micro-runtime/pull/3362 is merged.